src/booleans.js

```diff
@@ -11,6 +11,8 @@ const either = (a, b) => {
 };
 
 const none = (a, b) => {
+  // you can just return the expression.
```

Comment:
The `a === false && b === false` expression will always evaluate to either a truthy or a falsy value, so you can just return it - you don't need to return true/false:

```js
return a === false && b === false
```